Dockerfile

```diff
@@ -6,7 +6,7 @@ ENV DEBIAN_FRONTEND=noninteractive \
   COCOAPODS_ALLOW_ROOT=1 \
   CARGO_HOME=/root/.cargo \
   RUSTUP_HOME=/root/.rustup \
-  PATH=${PATH}:/root/.cargo/bin:/usr/lib/dart/bin
+  PATH=${PATH}:/root/.cargo/bin:/tmp/flutter/bin
```

Comment:
The `bin` folder contains `flutter` and `dart` already.

Reply:
Nice